src/app/workflow/permissions/permissions.component.ts

```diff
@@ -100,9 +97,8 @@ export class PermissionsComponent implements OnInit {
     }
   }
 
-  private handleError(e: HttpErrorResponse, defaultMessage: string) {
+  private handleError(e: HttpErrorResponse) {
```

Comment:
Why did LGTM suggest you change this one? Oh, because we weren't using `message` from line 105, which means we were ignoring the `defaultMessage` parameter?

Reply:
Yeah, we were never making use of the default message so got rid of it